Xamarin.Forms.Platform.UAP/Xamarin.Forms.Platform.UAP.csproj

```diff
@@ -256,6 +259,10 @@
       <SubType>Designer</SubType>
       <Generator>MSBuild:Compile</Generator>
     </Page>
+    <Page Include="PromptDialog.xaml">
```

Comment:
This won't be packaged unless it's added to the nuspec.

Reply:
Thanks...I didn't realize that I'll take a look.